src/Database/Adapter/Mongo/MongoDBAdapter.php

```diff
@@ -1016,7 +1015,7 @@ public function getSupportForCasting(): bool
     public function getNamespace(): string
     {
         if (empty($this->namespace)) {
-            throw new Exception('Missing namespace');
+            throw new DatabaseException('Missing namespace');
```

Comment:
Lets revert the changes in this file. Namespace, & missing database are not under the end user's control